strapiExperiment/testreact/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# pull official base image
+FROM node:13.12.0-alpine
```

Comment:
Let's make sure at one point the node version we use locally can be replicated in OCP